src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+
+    }
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
The toString() method could be simplified by using String.format() or concatenation instead of StringBuilder for such a simple case. StringBuilder is more efficient for complex string operations or when strings are being modified in loops, but here it just adds unnecessary complexity.

Reply:
Added return String.format